### PR TITLE
Project atomic blog: Increase the Base device size on docker daemon restart

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @jzb

**Comment by jzb** [comment]
@shishir-a412ed You'll want to ping @jberkus and @bproffitt in the future. They're handing the blog these days. I can put things up as backup if they're out, but they're running the show right now.

**Comment by shishir-a412ed** [comment]
@jzb Thanks !
ping @jberkus @bproffitt .

**Comment by jberkus** [comment]
@shishir-a412ed there are several formatting issues with the post, which I'll fix.   However, please address the line comments and the questions below:

1. Is this feature merged into docker or our docker extensions?  Where do people get it/try it?
2. Are you looking for feedback on the feature/proposal?  If so, how do people give that feedback?
3. Maybe a conclusion sentence to the post?

File: source/blog/2016-03-27-daemon_option_basedevicesize.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+ 
+# <span style="color:cornflowerblue">Increase the base device size on docker daemon restart</span>
+## <span style="color:cornflowerblue">Author: Shishir Mahajan</span>
+---
+Sometime back I wrote a feature for docker to allow expanding Base device size on daemon restart. Before we jump further into this article, I would like to point out that this feature is only available for devicemapper storage and does not apply to other storage drivers like overlay, btrfs, aufs etc.
```
Comment (jberkus) [inline source/blog/2016-03-27-daemon_option_basedevicesize.md]: Please provide a link to this feature/post.

**Comment by jberkus** [comment]
Also:

4. Can you compare this solution with adding volumes to your containers?  At least one sentence.

File: source/blog/2016-03-27-daemon_option_basedevicesize.md (hunk 1 above)
Comment (shishir-a412ed) [inline source/blog/2016-03-27-daemon_option_basedevicesize.md]: https://github.com/docker/docker/pull/19123

**Comment by jberkus** [comment]
Replaced by  PR #285

**Pull request closed without merging**